vta/hardware/intel/Makefile

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Directories
+ROOTDIR = $(CURDIR)
+BUILD_NAME = build
+BUILD_DIR = $(ROOTDIR)/../../$(BUILD_NAME)/hardware/intel
+SCRIPT_DIR = $(ROOTDIR)/scripts
+SRC_DIR = $(ROOTDIR)/../chisel
+
+# Debug flag
+DEBUG = false
+# Prevent generation of DSP
+NO_DSP = true
+# Device
+DEVICE = 5CSEBA6U23I7

```

Comment:
Can we derive this from invoking  `vta_config.json`: https://github.com/dmlc/tvm/blob/master/vta/config/vta_config.py#L94

See how it can be invoked in a Makefile: https://github.com/dmlc/tvm/blob/master/vta/hardware/xilinx/Makefile#L33

And also where to set the device string in `pkg_config.py`: https://github.com/dmlc/tvm/blob/master/vta/python/vta/pkg_config.py#L113
(you can ignore many of the pkg_config parameters since they are Xilinx specific)...

Reply:
Both DEVICE and DEVICE_FAMILY has been defined in the [pkg_config.py](https://github.com/dmlc/tvm/pull/3494/files#diff-842ae8b85a7944f55efd432de2d10994), parameterized in [Makefile](https://github.com/dmlc/tvm/pull/3494/files#diff-81af3620e4013b66642df37ed1408010) and [soc_system.tcl](https://github.com/dmlc/tvm/pull/3494/files#diff-b87eb483891e96c914c4464557ba1871).